clients/fides-js/src/lib/consent-types.ts

```diff
@@ -14,7 +15,6 @@ import type {
   TCFVendorLegitimateInterestsRecord,
   TCFVendorRelationships,
 } from "./tcf/types";
-import { CookieKeyConsent } from "~/lib/cookie";
```

Comment:
again, causing a build warning because of the absolute path. by importing `type` instead of the import, we can also get around the circular import